README.md

```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
 
 Axon is compatible with [Ethereum](https://ethereum.org) so that all of the develop utilities can be used on it directly. And the [Axon CLI](https://github.com/axonweb3/axon-cli) provides an all-in-one client which includes initialization, DevOps, cross-chain request and so on. Seeing is believing, there is a [15 minutes tutorial](https://docs.axonweb3.io/getting-started/for-dapp-devs/zero_to_axon_with_axon_cli) that will lead you to build your own chain and deploy a tiny application.
 
-### Native Cross-chain Communication
+### Native Cross-Chain Communication
 
 Openness and mobility are the foundation of social development, so is blockchain. Cross-chain function enhances liquidity for the web3 ecosystem. Axon develops native cross-chain communication without any bridge. Each Axon-based chain can speak to [CKB](https://www.nervos.org), other Axon-based chains, and any [IBC](https://ibcprotocol.org) compatible chains. Axon will embed more cross-chain protocols in the future.
```

Comment:
"Openness and mobility are the foundation of social development, so is blockchain." --> openness/mobility & blockchain are two different categories, trying other analogies?

_thoughts:
Openness and mobility are the foundation of social development, and technologies like blockchain where cross-chain function facilitates interoperability drives social changes. As a new cross-chain initiative, Axon is developing xxxx._

Reply:
Agreed with some changes to the opening sentence "Openness and mobility are the foundation of social development, so is blockchain." --> I guess we are trying to convey that openness are mobility are also the foundation of blockchain? It is not particularly clear.

Reply:
From my perspective, social development & blockchain dont fall into the same category either. Social development is a terminology/ theory explaining the changes within the society as a whole, yet blockchain is a technology, one of the factors that is driving the social change.

Reply:
Got it! Thank you!